pkg/api/steve/supportconfigs/supportconfigs.go

```diff
@@ -0,0 +1,57 @@
+package supportconfigs
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/rancher/rancher/pkg/version"
+)
+
+// supportConfig represents the serializable struct of support related information common to all rancher setups
+type supportConfig struct {
```

Comment:
Is there any reason the types we already had weren't ported over to this? https://github.com/rancher/csp-adapter/blob/release/v2.6/pkg/types/supportconfig.go